BrainPortal/db/migrate/20170220152128_signups_tweaks.rb

```diff
@@ -5,6 +5,13 @@ def up
     add_column    :signups, :user_id, :integer
     add_column    :signups, :hidden, :boolean, :default => false
 
+    Signup.all.each do |d|
+      user = User.where(:login => d.login).first
+      if user.present?
+        user[:signup_id] = user.id
```

Comment:
How can this even work? There is no signup_id in the user model! Have you even TESTED that code?